### PR TITLE
added support for custom networks

### LINKED ISSUE #51: Question: Egress IPAM operator within Azure
Hello,

First of all I would like to thank you for adding Azure support to this operator. We have had plans for quite some time to implement this but had to wait for Azure support.

In the documentation the following is present:
> Differently from AWS, in Azure networks can span multiple AZs and that is the way OCP is installed by default. Currently support for Azure allows you to deploy EgressIPs on the default vnet: `<infra-id>-vnet`

Can you share a bit more information on what's ment by 'the default vnet'?
Within our OpenShift setup we use seperate vnets for our master and worker vm's.
So our vnet name does not meet the `<infra-id>-vnet` that's listed in the documentation.

Does that mean we cannot use this operator in it's current state or am I misinterpreting the documentation?

Thank you in advance!

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
Signed-off-by: raffaelespazzoli <raffaele.spazzoli@gmail.com>

this will provide better support when the vnet already exists and specifically in UPI scenarios.

Fixes #51

### CODE REVIEW

File: config/crd/bases/_.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (sabre1041) [inline config/crd/bases/_.yaml:1]: What is the purpose of this file?

**Review by sabre1041 — COMMENTED** [review]
Looks ok. One comment

File: config/crd/bases/_.yaml (hunk 1 above)
Comment (raffaelespazzoli) [reply config/crd/bases/_.yaml:1]: it's an issue of the operator sdk. I'm am chatting with the guys who are supposed to help me fix it. not sure what to say it gets generated ...

Comment (sabre1041) [reply config/crd/bases/_.yaml:1]: If it gets generated, please include in a `.gitignore` file

Comment (raffaelespazzoli) [reply config/crd/bases/_.yaml]: done

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**